packages/admin-panel/src/pages/resources/EntitiesPage.js

```diff
@@ -65,7 +65,7 @@ const IMPORT_CONFIG = {
       },
     },
     {
-      label: 'Automatically fetch GeoJSON (defaults to "Yes")',
+      label: 'Automatically fetch GeoJSON (defaults to "Yes", requires parent_code)',
```

Comment:
Happy to remove this if too cluttered but auto geoJson fetch will fail with no parent_codes
![image](https://user-images.githubusercontent.com/62525113/132810678-904f1b14-f609-4bf2-98b3-b36bc5e4a537.png)

Reply:
Yeah I'd leave off and have it throw an error that can show up here instead